cmd/s3-gw/app.go

```diff
@@ -483,10 +460,6 @@ func (a *App) configReload() {
 		return
 	}
 
-	if err := a.bucketResolver.UpdateResolvers(a.getResolverConfig()); err != nil {
```

Comment:
Same thing as with HTTP, we still need a way to change config in runtime.

Reply:
returned